src/logql.grammar

```diff
@@ -356,7 +357,7 @@ ConvOp {
     (std.asciiLetter | "_" | ":") (std.asciiLetter | std.digit | "_" | ":" )* 
   }
 
-    LabelName { (std.asciiLetter | "_") (std.asciiLetter | std.digit | "_")* }
```

Comment:
Just checking: is this a wanted change?

Reply:
Yep, that's just removing unnecessary indentation.